services/elastic_ip_manager/src/main/resources/application.properties

```diff
@@ -47,7 +47,7 @@ management.health.redis.enabled=false
 rbac.policy.type=Enforced
 
 ##### Jaeger ####
-opentracing.jaeger.udp-sender.host=localhost
+opentracing.jaeger.udp-sender.host=10.111.132.59

```

Comment:
Remove all "^M"

Reply:
Do you notice the ending character of each line, which is "^M"? Could you remove them.

Reply:
Yes, in the latest file changes the "^M" are been removed

Reply:
Can you check https://github.com/futurewei-cloud/alcor/pull/675/files, which shows that there was no  change to this application.properties?

Reply:
I changed it back from "10.111.132.59" to "localhost" (the original code). Overall there is no change to the application.properties file :)

Reply:
This remains unresolved.

Reply:
Sorry about the miss understanding. Should be fixed now.